services/stats/stats.go

```diff
@@ -35,6 +36,9 @@ var enableGCStats bool
 var rc runtimeStatsCollector
 var pkgLogger logger.LoggerI
 var statsSamplingRate float32
+var isMockClient bool
```

Comment:
I propose to group all tag-related variables together. Also some renaming might be appropriate. What I have in mind is:
```
// place all variables in a group
taggedClientsMapLock // locks should be above the resources they are protecting
taggedClientsMap
tagValsList // rename to taggedClientKeys
tagStrList // rename to taggedClientTags
isMockClient // rename to connEstablished and adapt logic with the reversal of this boolean
```

Reply:
We can also place everything inside a `var` blog:

```go
var (

// place all variables in a group
taggedClientsMapLock // locks should be above the resources they are protecting
taggedClientsMap
tagValsList // rename to taggedClientPendingKeys
tagStrList  // rename to taggedClientPendingTags
isMockClient // rename to connEstablished and adapt logic with the reversal of this boolean
)
```

Reply:
done